owl/SOMA-HOME.owl

```diff
@@ -385,6 +432,21 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-HOME.owl#TrashContainer -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-HOME.owl#TrashContainer">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#DesignedContainer"/>
```

Comment:
I would put the containment disposition to the designedcontainer class instead.

Are there any trash-container specific dispositions? This question also @daniel86 

For example, trash containers often -- but not always -- have some sort of lid or other covering mechanism built in. This is because they are intended to prevent smells from their contents from readily propagating and/or limit access of insects to the trash.